src/DynamoCoreWpf/ViewModels/Watch3D/DynamoLineGeometryModel3D.cs

```diff
@@ -13,6 +13,17 @@ protected override SceneNode OnCreateSceneNode()
         {
             return new DynamoLineNode();
         }
+
+        /// <summary>
+        /// This method is used to set the dynamo interaction state into an existing material slot
+        /// which is sent to the shader buffer for lines.
+        /// </summary>
+        /// <param name="state"></param>
+        internal void SetState(int state)
+        {  
+             //TODO should we also use this for points if it works for lines to make them consistent?
+            this.material.FadingNearDistance = state;
```

Comment:
Did you replace this for `blendingfactor` as it is common to both point and line and because `blendingfactor` is missing for line?

Reply:
yep yep.